examples/fit_with_bounds.py

```diff
@@ -18,21 +18,32 @@
 init_logging(level="debug")
 np.random.seed(15816)
 # {{{ helper function(s)
-class myfitclass (object):
-    def __init__(self):
+class myfitclass (nddata):
+    def __init__(self,*args,**kwargs):
+        # copied from fitdata
         self.expression = None
-    expression = (2+2)
-    def gen_from_expr(self,data, guesses={}):
+        if isinstance(args[0],nddata):
+            # move nddata attributes into the current instance
+            myattrs = normal_attrs(args[0])
+            for j in range(0,len(myattrs)):
+                self.__setattr__(myattrs[j],args[0].__getattribute__(myattrs[j]))
+        else:
+            nddata.__init__(self,*args,**kwargs)
+        return
+    @property
+    def functional_form(self):
```

Comment:
this function has the same name as the function below it? won't this cause issues?

Reply:
No — the decorator on the line before is different